src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -679,5 +682,6 @@ div.account-selector
 .loader-container
   position: relative
   height: 80px
-
+.amount-credit-curr
+  margin: auto
```

Comment:
I didn't make the changes on the send money side because i don't have access to the currency name in recipient object, i'm waiting for @vaab to make the changes on the object.

Reply:
i'll do it as soon as @vaab provide it

Reply:
👍 
Adding the WIP label...

Reply:
@seegwen @vaab the currency label applies for send and ask money now

Reply:
👍